code/modules/power/singularity/field_generator.dm

```diff
@@ -220,6 +221,7 @@ no power level overlay is currently in the overlays list.
 /obj/machinery/field/generator/proc/turn_on()
 	active = FG_CHARGING
 	addtimer(CALLBACK(src, PROC_REF(warm_up)), 5 SECONDS)
+	ADD_TRAIT(loc, TRAIT_CONTAINMENT_FIELD, REF(src))
```

Comment:
This trait needs to be updated in `Moved`, applying it to the new loc and removing it from the old, despite how unlikely it may seem that it moves when active. 
(`qdel`ing moves you to nullspace, as an example)